vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -28,14 +28,11 @@ packages:
      - lxml
     pip3:
      - virtualbmc
-     - python-ironicclient
-     - python-ironic-inspector-client
      - lxml
      - netaddr
      - requests
      - setuptools
      - libvirt-python
-     - python-openstackclient
```

Comment:
I don't have an ubuntu box to test this on - anyone know if these other deps were added for the clients?  Seems likely but I don't want to remove them without testing :)

Reply:
Ok so I did some testing on a freshly installed RHEL8 box, and it seems the lxml, netaddr and libvirt-python dependencies are required for ansible - we can probably remove some of these though (certainly virtualbmc and on RHEL I removed crudini, pip, requests, setuptools and nodejs)

Reply:
virtualbmc can be removed, probably requests and setuptools too I don't think they are used anywhere else. I don't like to say this, but if you don't have an ubuntu box and nobody else tries it, the CI would anyways complain if something major is broken when running /test-integration

Reply:
Ok thanks, I'm testing locally on CentOS7 (not sure why the test-centos-integration didn't trigger?) and will try to find time to test on Ubuntu tomorrow after adjusting the package removals to match.